server/service/src/backend_plugin/plugin_provider.rs

```diff
@@ -1,28 +1,23 @@
 use std::sync::{Arc, RwLock};
 
-use actix_web::web::Data;
-
 use base64::{prelude::BASE64_STANDARD, Engine};
 
 use repository::{BackendPluginRow, FrontendPluginRow, PluginType, PluginTypes, PluginVariantType};
 use serde::{de::DeserializeOwned, Deserialize, Serialize};
 use thiserror::Error;
 
-use crate::{backend_plugin::boajs, service_provider::ServiceProvider};
-
-use super::boajs::BoaJsPluginError;
+use crate::boajs::{self, BoaJsError};
 
 #[derive(Debug, Error, PartialEq)]
-#[error("Error in plugin {r#type:?}")]
+#[error("Error in plugin {code}")]
```

Comment:
One thing i've changed in this file, is holding plugin code within PluginInstanceVariant, since i really wanted to show plugin code on error